icon/Icon.js

```diff
@@ -206,6 +211,8 @@ export const Icon = {
     paste(p)            {return Icon.icon({...p,  iconName: 'paste'})},
     pause(p)            {return Icon.icon({...p,  iconName: 'pause'})},
     pauseCircle(p)      {return Icon.icon({...p,  iconName: 'pause-circle'})},
+    person(p)           {return Icon.icon({...p,  iconName: 'user'})},
```

Comment:
I would rather we just go ahead and make this one `user()` and make the existing-one `userCircle()`.